src/ood-preview/lib/ood_preview/utils.ml

```diff
@@ -0,0 +1,33 @@
+open Import
+
+let extract_metadata_body s =
+  match Str.split (Str.regexp "---") s with
```

Comment:
We could use [jekyll-format](https://github.com/avsm/jekyll-format) for this but seems this is working haha :))

Reply:
Oh nice! I didn't know about jekyll-format. The function is really hacky, so I'm happy to replace it :)
jekyll-format comes with a lot of new dependencies for a single function though, so maybe we can simply copy the relevant bits? (https://github.com/avsm/jekyll-format/blob/master/src/jekyll_format.ml#L17)